### PR TITLE
Fix daemon and python 3.7 issues v2

### PR DESCRIPTION
**Pull request by gonzoleeman**
I believe this pull request fixes all issues raised in the first version:
* removed reference to not-yet-present targetcli.service file
* moved "to.sock = sock" earlier
* combined 2 sets of 2 commits that were related
* cleaned up some merge errors of my own that I found

### CODE REVIEW

File: daemon/targetclid [hunk 1]
```diff
@@ -202,35 +202,47 @@ def main():
...
+            pass
+
+        # Create a TCP/IP socket
+        try:
+            sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+        except socket.error as err:
+            to.display(to.render(err.strerror, 'red'))
+            sys.exit(1)
+
+        # save socket so a signal can clea it up
```
Comment (pkalever) [inline daemon/targetclid]: Same here:
s/clea/clean ?

File: daemon/targetclid [hunk 2]
```diff
@@ -202,35 +202,47 @@ def main():
...
+    if getenv('LISTEN_PID'):
+        # the systemd-activation path, using next available FD
+        fn = sys.stderr.fileno() + 1
+        try:
+            sock = socket.fromfd(fn, socket.AF_UNIX, socket.SOCK_STREAM)
+        except socket.error as err:
+            to.display(to.render(err.strerror, 'red'))
+            sys.exit(1)
+
+        # save socket so a signal can clea it up
```
Comment (pkalever) [inline daemon/targetclid]: minor one:
s/clea/clean ?

May be Maurizio can fix it when merging.

File: daemon/targetclid (hunk 1 above)
Comment (gonzoleeman) [reply daemon/targetclid]: I'd rather see this fixed with a subsequent commit, since it's not functional.

File: daemon/targetclid (hunk 2 above)
Comment (gonzoleeman) [reply daemon/targetclid]: Same as the other one: I'd rather fix with a follow-on commit.

**Comment by gonzoleeman** [comment]
Would you like me to fix the spelling errors in an additional commit?

**Comment by pkalever** [comment]
> Would you like me to fix the spelling errors in an additional commit?

This is not important, for projects which I maintain, I generally fix it while merging, avoiding the need to respin for the author/contributor :-)

Or 

IMHO, we can fix it later when we come across that part of code in our future PR's.

Thanks!

**Comment by maurizio-lombardi** [comment]
Looks good to me, thanks @gonzoleeman

**Pull request merged**